common/src/main/java/com/wynntils/features/ui/BulkBuyFeature.java

```diff
@@ -30,7 +30,7 @@
 import net.minecraftforge.eventbus.api.EventPriority;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
-@ConfigCategory(Category.INVENTORY)
+@ConfigCategory(Category.UI)
```

Comment:
I think neither Ui nor Inventory really fits. If it is important to you, I can accept the change to UI. (But I think renaming Trademarket to Trading or Trade or Buying or something like that, and placing this there as well, would make more sense)